domain/test/usecases/received/copy_multiple_files_to_my_space_from_received_shares_interactor_test.dart

```diff
@@ -0,0 +1,112 @@
+// http://www.linshare.org, between linagora.com and Linagora, and (iii) refrain from
+// infringing Linagora intellectual property rights over its trademarks and commercial
+// brands. Other Additional Terms apply, see
+// <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf>
+// for more details.
+// This program is distributed in the hope that it will be useful, but WITHOUT ANY
+// WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for
+// more details.
+// You should have received a copy of the GNU Affero General Public License and its
+// applicable Additional Terms for LinShare along with this program. If not, see
+// <http://www.gnu.org/licenses/> for the GNU Affero General Public License version
+//  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
+//  the Additional Terms applicable to LinShare software.
+//
+
+import 'package:dartz/dartz.dart';
+import 'package:domain/domain.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:testshared/testshared.dart';
+
+import '../../mock/repository/authentication/mock_document_repository.dart';
+
+void main() {
+  group('copy_multiples_files_to_my_space_from_received_shares_interactor tests', () {
+    MockDocumentRepository documentRepository;
+    CopyToMySpaceInteractor copyToMySpaceInteractor;
+    CopyMultipleFilesFromReceivedSharesToMySpaceInteractor _copyMultipleFilesFromReceivedSharesToMySpaceInteractor;
+
+    setUp(() {
+      documentRepository = MockDocumentRepository();
+      copyToMySpaceInteractor = CopyToMySpaceInteractor(documentRepository);
+      _copyMultipleFilesFromReceivedSharesToMySpaceInteractor = CopyMultipleFilesFromReceivedSharesToMySpaceInteractor(copyToMySpaceInteractor);
+    });
+
+    test('copy multiples files to my space from received shares interactor should return success with valid data', () async {
+      when(documentRepository.copyToMySpace(
+          CopyRequest(receivedShare1.shareId.uuid, SpaceType.receivedShare)))
+      .thenAnswer((_) async => [document1]);
+
+      when(documentRepository.copyToMySpace(
+          CopyRequest(receivedShare2.shareId.uuid, SpaceType.receivedShare)))
+      .thenAnswer((_) async => [document2]);
+
+      final result = await _copyMultipleFilesFromReceivedSharesToMySpaceInteractor.execute(shares: [receivedShare1, receivedShare2]);
+      final resultsList = result
+          .map((success) => (success as CopyMultipleToMySpaceFromReceivedSharesAllSuccessViewState).resultList)
+          .getOrElse(() => []);
+      expect(resultsList, [Right<Failure, Success>(CopyToMySpaceViewState([document1])), Right<Failure, Success>(CopyToMySpaceViewState([document2]))]);
+    });
+
+    test('copy multiples files to my space from received shares interactor should return success with some failures', () async {
```

Comment:
This case `copy multiples files to my space from received shares interactor should return success with some failures` and case `copy multiples files to my space from received shares interactor should fail with only failures` are quite versus.